lib/diagnostic.rb

```diff
@@ -93,13 +93,13 @@
 # In a Ruby string, write how you would remove the last two elements from this
 # array
 
-Diagnostic.remove_from_array = 'your answer here'
+Diagnostic.remove_from_array = 'arr.delete_if {|num| num > 50 }'
 
```

Comment:
Nice!! This is one way of doing it! 
You could also do `arr.pop 2` (which you will probably see more often in the wild)